.circleci/config.yml

```diff
@@ -1,115 +1,163 @@
 version: 2.1
 orbs:
-  android: circleci/android@1.0
+  android: circleci/android@2.1.2
```

Comment:
Are you able to upgrade the Android orb to 2.4.0?

Reply:
I was having spotless not found error when I switch it to 2.4.0.  I will have another PR with Gradle changes and will include it there.